src/setup_commands/defaults_commands.rs

```diff
@@ -670,7 +674,7 @@ async fn standard(ctx: &Context, msg: &Message, _: Args) -> CommandResult {
 async fn match_win_points(ctx: &Context, msg: &Message, mut args: Args) -> CommandResult {
     use squire_lib::settings::StandardScoringSetting::*;
     let data = ctx.data.read().await;
-    match args.single_quoted::<f64>() {
+    match args.single_quoted::<r64>() {
```

Comment:
It should be the case that this correctly parses strings like `1`, `0.5`, and `-2`, but I'm not sure. Did you test this?

Reply:
I followed `squire_lib` instead of testing myself. I will test it.